fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/common/StringUtils.java

```diff
@@ -174,6 +174,18 @@ public String string2Vector(String text, String vectorName) {
 		return sb.toString();
 	}
 	//===========================================================
+	/*
+	 * Escape double quotes in a string 
+	 */
+	//===========================================================
+	public String escapeQuotes(String text) {
+            return text.replace("\"", "\\\"");
+	}
+	//===========================================================
```

Comment:
[Minor] This comment is double, can be delete.
```java

/**
 * Comment a String with more than one line
*/
//===========================================================
```

Reply:
Done thanks for pointing it out.